lib/src/models.dart

```diff
@@ -1,11 +1,16 @@
 export 'api_exception.dart' show APIException;
-export 'book.dart' show Book;
-export 'book_title.dart' show BookTitle;
-export 'comment.dart' show Comment;
-export 'image.dart' show Image;
-export 'image_type.dart' show ImageType;
-export 'search.dart' show Search, SearchSort;
-export 'tag.dart' show Tag;
-export 'tag_type.dart' show TagType;
-export 'tags_list.dart' show TagsList;
-export 'user.dart' show User;
+export 'models/book.dart' show Book;
+export 'models/book_images.dart' show BookImages;
+export 'models/book_title.dart' show BookTitle;
+export 'models/comment.dart' show Comment;
+export 'models/image/cover.dart' show Cover;
+export 'models/image/cover_thumbnail.dart' show CoverThumbnail;
+export 'models/image/image.dart' show Image;
+export 'models/image/image_thumbnail.dart' show ImageThumbnail;
+export 'models/image/image_type.dart' show ImageType;

```

Comment:
That can be made into `models/image.dart`.

Reply:
Also, better name folder `images`, than `image`. And it will be `models/images.dart` then.